src/tools/environment.js

```diff
@@ -31,10 +31,29 @@ const cleanEnvironment = () => {
   }
 };
 
+const localFilepath = filename => {
+  return path.join(process.cwd(), filename || '');
```

Comment:
they couldn't, but they also couldn't before. In [master](https://github.com/zapier/zapier-platform-core/blob/master/src/tools/environment.js#L36), we resolved their filename to the local directory anyway. We could expand that, but that seems like a more disruptive change (instead of renaming, they have to pass the whole path in).